django_project/certification/models/certificate_type.py

```diff
@@ -0,0 +1,55 @@
+"""Certificate type model for certification app"""
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from base.models.project import Project
+
+
+class CertificateType(models.Model):
+    name = models.CharField(
+        help_text=_('Certificate type.'),
+        max_length=200,
+        null=False,
+        blank=False,
+        unique=True,
+    )
+
+    description = models.TextField(
+        help_text=_('Certificate type description - 1000 characters limit.'),
+        max_length=1000,
+        null=True,
+        blank=True,
+    )
+
+    wording = models.CharField(
+        help_text=_(
+            'Wording that will be placed on certificate. '
+            'e.g. "Has attended and completed".'
+        ),
+        max_length=500,
+        null=False,
+        blank=False
+    )
+
+    order = models.IntegerField(
+        blank=True,
+        null=True,
+        unique=True
+    )
+
+    def __str__(self):
+        return self.name
```

Comment:
check codecov:
This one is covered in test_CRUD_CertificateType
https://github.com/kartoza/prj.app/pull/1383/files#diff-dd4bdce4503bf9fafe9947650eb5d398f8c1c897642b377dec9ef0847eb8c8e0R145